test/harness/e2e/vm/domain-template.xml

```diff
@@ -24,7 +24,9 @@
     <boot dev='hd'/>
     <firmware>
       <feature enabled='yes' name='secure-boot'/>
-      <feature enabled='yes' name='enrolled-keys'/>
+      <!-- we enable secureboot to make sure that attestation is happening, but don't enroll the keys to avoid
+	   issues booting vms in fedora -->
+      <feature enabled='no' name='enrolled-keys'/>
```

Comment:
Should we make this conditional and on by default? Could be a follow-up.

Reply:
it's ok like this, we don't really need the signature check for our testing.